### PR TITLE
site: add blog page

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add a blog page to our website for posting newsletters and articles

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
site: add blog page 
```
image

### CODE REVIEW

**Comment by zirain** [comment]
prefer to call these as `Announcement` or `Release Announcement`

File: docs/content/en/blog/0.2.0_release.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Kurator v0.2.0 is live!"
+date: 2023-02-09
+linkTitle: "Kurator v0.2.0 is live!"
+---
+
+Kurator v0.2.0 adds observability and cluster lifecycle management, featuring multi-cluster monitoring with Thanos, real-time K8s app monitoring with Pixie, and lifecycle management for local data center and AWS-built clusters, enhancing distributed cloud-native infrastructure management.
```
Comment (hzxuzhonghu) [inline docs/content/en/blog/0.2.0_release.md:7]: Add a link to the detailed note

Comment (Xieql) [reply docs/content/en/blog/0.2.0_release.md:7]: Apologies for not being clear enough. 

The purpose of this PR is to create a blog page, and the current articles are just placeholders to demonstrate the blog layout. 

I will submit complete articles later to replace these placeholder posts. 

This is the full plan https://github.com/kurator-dev/kurator/issues/452

**Comment by Xieql** [comment]
> prefer to call these as `Announcement` or `Release Announcement`

Apologies for not being clear enough. 

The purpose of this PR is to create a blog page, and the current articles are just placeholders to demonstrate the blog layout. 

I will submit complete articles later to replace these placeholder posts. 

This is the full plan https://github.com/kurator-dev/kurator/issues/452

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**